src/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -718,6 +718,13 @@ private function registerElasticsearchConfiguration(ContainerBuilder $container,
             return;
         }
 
+        $clientClass = class_exists(\Elasticsearch\Client::class) ? \Elasticsearch\Client::class : \Elastic\Elasticsearch\Client::class;
+
+        $clientDefinition = new Definition($clientClass);
+        $clientDefinition->setPublic(false);
```

Comment:
you don't need that or do you?

Reply:
Just did it as it was before: https://github.com/api-platform/core/pull/5503/files#diff-e55025391dae392739748b2ff664550ac2bb4b800005b9f22da70fb43a8eec4bL8